lib/bloc/account/account_bloc.dart

```diff
@@ -234,6 +234,9 @@ class AccountBloc extends Cubit<AccountState> with HydratedMixin {
           (amount > accState.maxInboundLiquidity && amount <= channelMinimumFee)) {
         throw PaymentBelowSetupFeesError(channelMinimumFee);
       }
+      if (amount > accState.maxInboundLiquidity) {
```

Comment:
@erdemyerebasmaz  This will seems like an error to me. It will throw on the normal case where fees are not null and the user is trying to receive more than the inbound liquidity. We only need to throw in that case IMO if the fees are null.

Reply:
@roeierez That's right, thought I put this under `else if`. 
Addressed d4d9667e01efb9fbf8454bdfb47112f45a72293a.